src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -286,8 +286,6 @@ cli_enable_maintenance(int argc, char **argv)
 	bool pgIsNotRunningIsOk = true;
 	bool monitorDisabledIsOk = false;
 
-	char *channels[] = { "state", NULL };
-
```

Comment:
Here we are undoing PR #146

Reply:
Good catch! I need to review that closely, thanks.